core/src/main/java/com/datastax/oss/driver/internal/core/loadbalancing/BasicLoadBalancingPolicy.java

```diff
@@ -122,7 +121,7 @@ public class BasicLoadBalancingPolicy implements LoadBalancingPolicy {
   private volatile NodeDistanceEvaluator nodeDistanceEvaluator;
   private volatile String localDc;
   private volatile NodeSet liveNodes;
-  private final List<String> preferredRemoteDcs;
+  private final LinkedHashSet<String> preferredRemoteDcs;
```

Comment:
Switch to just modeling preferredRemoteDcs as an (ordered) Set suggested by @adutra